### DISCUSSION TITLE
Has Shuffle feature been removed?

### DISCUSSION THREAD

**Discussion by albedor** [t1m0thyj/WinDynamicDesktop#571] [Q&A]
I remember there being a shuffle feature in the "more options" tab, but I can no longer find it after reinstalling WinDyanmicDesktop on my new laptop. Was it removed, or am I just unable to see it? I tried both the Microsoft store download and the GitHub download, but both have not yielded anything. Here is a screenshot of my options menu: 
![WinDynamic](https://github.com/t1m0thyj/WinDynamicDesktop/assets/102763868/8bef8aad-3f57-4b69-903d-2b9c7f54371b)

**Comment by t1m0thyj** [comment]
The features to "Shuffle themes daily" and "Change lockscreen image" still exist but were moved to a new location in 5.4.0 to declutter the tray menu. They can now be found by clicking the "meatballs" menu in the Select Theme dialog:

![image](https://github.com/t1m0thyj/WinDynamicDesktop/assets/22344007/c4bc203e-ce22-471e-8d49-8220653b603b)

**Note:** There was a bug present in 5.4.1 and fixed in 5.4.2 where the 3 dots icon was hidden on high DPI displays.

**Reply by iBNzTTAi** [reply]
Changing the lock screen image is not as user-friendly and convenient as before

**Reply by t1m0thyj** [reply]
@BellModena Thanks for the feedback. I understand that the new setting is less convenient to access then the old location where it was accessible in the system tray menu. However I believe the change is worthwhile because now there are more options available to synchronize the lock screen image with any display, not just the primary one.
